### PR TITLE
Enable CD

### PR DESCRIPTION
**Pull request by jglick**
https://www.jenkins.io/doc/developer/publishing/releasing-cd/
https://github.com/jenkins-infra/repository-permissions-updater/pull/2637

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -4,40 +4,35 @@
...
         <jenkins.version>2.303.3</jenkins.version>
+        <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
     </properties>
 
-    <developers>
-        <developer>
-            <id>vjuranek</id>
-            <name>Vojtech Juranek</name>
-        </developer>
-    </developers>
```
Comment (jglick) [inline pom.xml:26]: https://github.com/jenkins-infra/repository-permissions-updater/pull/2637#issuecomment-1180008909

File: pom.xml [hunk 2]
```diff
@@ -4,40 +4,35 @@
...
+        <tag>${scmTag}</tag>
     </scm>
 
     <dependencyManagement>
         <dependencies>
             <dependency>
                 <groupId>io.jenkins.tools.bom</groupId>
                 <artifactId>bom-2.303.x</artifactId>
-                <version>1382.v7d694476f340</version>
+                <version>1478.v81d3dc4f9a_43</version>
```
Comment (jglick) [inline pom.xml:35]: While I am here, to clean up the explicit `version` below.

**Review by vjuranek — APPROVED** [review]

**Comment by daniel-beck** [comment]
Secrets exist.

**Comment by timja** [comment]
> @timja FYI I have no admin permission on this repo so I cannot check whether the secrets got uploaded.

fixed

**Pull request merged**